2019-20/D01 Make Individual File.sps

```diff
@@ -72,6 +72,15 @@ Else if cij_pattype_code = 2.
     Compute CIJ_mat = Distinct_CIJ.
 End if.
 
+*Create variables to count preventable admissions and preventable beddays. 
+Do if cij_ppa.
+    If Distinct_CIJ preventable_admissions = 1.
+    Compute preventable_beddays = datediff(Max(!startFY, CIJ_start_date), Min(!endFY, CIJ_end_date), "days").
```

Comment:
These dates need swapping around as it is producing negative time-lengths at the moment.